Samples/Objective-C/nhubsample-refresh/nhubsample-refresh/SetupViewController.m

```diff
@@ -79,7 +79,7 @@ - (void)notificationHub:(MSNotificationHub *)notificationHub didReceivePushNotif
     dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(2.0 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
         [alertController dismissViewControllerAnimated:YES completion: nil];
     });
-    completionHandler((notification.data != nil && [notification.data count] > 0) ? UIBackgroundFetchResultNewData : UIBackgroundFetchResultNoData);
+    completionHandler(UIBackgroundFetchResultNoData);
```

Comment:
Need to fix swift sample as well

Reply:
Fixed in https://github.com/Azure/azure-notificationhubs-ios/pull/64/commits/1d26c8ef86e592ee39e5b61c91c7720453477444